docs/tool/manifest.js

```diff
@@ -11,35 +13,65 @@ const { kebabCase } = require( 'lodash' );
  *
  * @return {Array} manifest.
  */
-module.exports = function( parsedNamespaces, packagesConfig ) {
-	const dataManifest = [ {
-		title: 'Data Package Reference',
-		slug: 'data',
-		markdown_source: 'https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/data/README.md',
-		parent: null,
-	} ].concat(
-		Object.values( parsedNamespaces ).map( ( parsedNamespace ) => {
-			const slug = `data-${ kebabCase( parsedNamespace.name ) }`;
-			return {
-				title: parsedNamespace.title,
-				slug,
-				markdown_source: `https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/data/${ slug }.md`,
-				parent: 'data',
-			};
-		} )
-	);
-
-	const packagesManifest = Object.entries( packagesConfig ).map( ( [ folderName, config ] ) => {
-		const path = config.isNpmReady === false ?
-			`https://raw.githubusercontent.com/WordPress/gutenberg/master/${ folderName }/README.md` :
-			`https://raw.githubusercontent.com/WordPress/gutenberg/master/packages/${ folderName }/README.md`;
-		return {
-			title: `@wordpress/${ folderName }`,
-			slug: `packages-${ folderName }`,
-			markdown_source: path,
-			parent: 'packages',
-		};
-	} );
-
-	return packagesManifest.concat( dataManifest );
+module.exports = {
+	getPackageManifest: function( packagesConfig ) {
```

Comment:
Minor; I would have moved those functions above (and added JSDocs) and only export the `getManifest`concatenating the three manifests.